### PR TITLE
CODEOWNERS changes & renovate reviews

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by armenzg** [comment]
@sarah-clements @camd r?

File: .renovaterc [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@uirouter/angularjs"
   ],
   "labels": ["dependencies", "javascript"],
-  "reviewers": ["armenzg"],
+  "reviewers": ["sclements"],
```
Comment (sarah-clements) [inline .renovaterc:17]: Ah, you beat me to it. I had a pr for this but hadn't merged it yet :) This should be my github handle right? If so, it's 'sarah-clements'.

File: CODEOWNERS [hunk 2]
```diff
@@ -1,14 +1,8 @@
 # Visit https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 # for learning about the syntax
 
-/.travis.yml                @armenzg
-/app.json                   @armenzg
-/docker-compose.yml         @armenzg
```
Comment (sarah-clements) [inline CODEOWNERS:6]: Maybe we should keep some of these but change it to treeherder admins.

**Comment by armenzg** [comment]
Hi Sarah,
I've addressed your concerns. Could you please merge if it's satisfying?

Would you mind to give me some access to the repo? (e.g. assigning myself or even write pending admin merging)
I have not yet decided if I will contribute long term but I think it would not be too much concern if I had *some* access.

**Comment by sarah-clements** [comment]
> Hi Sarah,
> I've addressed your concerns. Could you please merge if it's satisfying?
> 
> Would you mind to give me some access to the repo? (e.g. assigning myself or even write pending admin merging)
> I have not yet decided if I will contribute long term but I think it would not be too much concern if I had _some_ access.

I've sent you an invite with write access. Any contributions you'd like to make would be most welcome :)

**Comment by armenzg** [comment]
Thank you Sarah!

**Pull request merged**